src/qtbase/QObject.js

```diff
@@ -40,3 +40,7 @@ function QObject(parent) {
     // http://doc.qt.io/qt-5/qtqml-javascript-dynamicobjectcreation.html
     this.destroy = this.$delete;
 }
+
+QObject.prototype.toString = function() {
+    return `${this.constructor.name}()`;
```

Comment:
Maybe add a `QObject.$toString(s)`, where "s" is put in between the parentheses? Save all the calls to "replace"?